active/0046-improved-release-process.md

```diff
@@ -0,0 +1,55 @@
+
+We propose that we adopt a pattern similar to Ember itself in order to align with the expectations of the Ember community, more-clearly communicate around our release lifecycle, and provide rigor around our support structure. Anything which is not specifically called out as a difference in this document is inferred to be following the patterns specified by Ember itself.
+
+## Channel Design
+
+To begin there will be three separate channels: canary, beta, and release. We intend to investigate an LTS channel after this process has matured.
+
+- **Canary**: represents the latest work in ember-cli, and is synonymous with the `HEAD` of the `master` branch and is the least stable of all channels.
+- **Beta**: branched off of master every six weeks, exact commit decided upon manually. Updated and released weekly with commits that are prefixed `[BUGFIX beta]`. Less stable than `release` as it is a proving ground. No new features will be added once the branch has been created to allow for existing features to mature.
+- **Release**: branched off of Beta every six weeks. Only rarely will this be updated, but possible for security issues and uncaught regressions.
+ 
+ember-cli will not support daily releases as time-based packaging doesn't make a lot of sense.
+
+## New Features
+
+New features to ember-cli must be protected by feature flags. Incomplete and WIP features will be available in the Canary channel, but will not be available in the Beta or release channels.
+
+## Tooling Design
+
+We must create additional tooling and patterns in order to make this efficient. Since ember-cli successfully installs and works from npm without modification we don't need to bundle and publish an asset for each Canary build. We'll publish tags to npm for `beta` and `release` channel releases so that they're not tied to a git remote URL. The `latest` tag (the default when installing via `npm install -g ember-cli`) will track our `release` channel at all times.
+
+## Timeline
+
+Since the ember-cli project is presently designed to track Ember development, we'll run our release schedule on a one week delay from Ember itself. This ensures that we're able to incorporate the latest changes from Ember into ember-cli and gives us a week to check for Ember-introduced regressions. As Ember itself becomes an npm module this will become less of a concern and we can diverge on our release schedule as best suits the ember-cli project. We will ship the last beta coincidentally with the newest Ember release.
```

Comment:
In general, I believe that we could do the same timeframe as Ember, but I think "within 1 week" gives us adequate leeway to do what works best.  The first few runs, I suspect it will take a bit more time and having that extra couple days might be nice...